site/en/docs/crux/api/index.md

```diff
@@ -235,6 +235,29 @@ Note: The values for each percentile are synthetically derived, it does not impl
 </tbody>
 </table></div>
 
+### Collection Period
+
+As of October 2022, the CrUX API contains a `collectionPeriod` object listing the `firstDate` and `endDate` included in the collection. An example is provided below:
+
+```json
+    "collectionPeriod": {
+      "firstDate": {
+        "year": 2022,
+        "month": 10,
+        "day": 9
+      },
+      "lastDate": {
+        "year": 2022,
+        "month": 9,
+        "day": 12
+      }
+    }
+```
+
+This allows better understanding of the data and whether it's been updated yet for that day or is returning the same data as yesterday.
+
+Note that CrUX is approximately 2 days behind today's date since it waits for completed data for the day, and there is some processing time involved before it is available in the API.
```

Comment:
```suggestion
Note that the CrUX API is approximately two days behind today's date since it waits for completed data for the day, and there is some processing time involved before it is available in the API.
```